note-new

```diff
@@ -1,18 +1,19 @@
 #!/bin/bash
 
-required_date="${1:-today}"
-
 rootFolder="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 rootFolder="$( cat "${rootFolder}/.noteFolderPath")"
+
 if [ -z $rootFolder ]; then
   echo "Can't find root folder '${rootFolder}', did you run note-setup?"
   exit 1
 fi
 
-formated_date="$( date -d "${required_date}" +'%Y-%m-%d' )"
 noteFolderLocation="${rootFolder}/notes"
 
-filePath="${noteFolderLocation}/${formated_date}.md"
+# parseInputDate (function) and formattedDate (variable) are sourced from date-parser
+source "${rootFolder}/date-parser" $1
```

Comment:
`source` are like import, move it to the top of the file

- Arguments
- Source
- Functions
- Code